src/menu.scss

```diff
@@ -5,352 +5,331 @@
+
+  // Menu Icon
+  $fd-menu-icon-size: var(--sapFontLargeSize) !default;
+  $fd-menu-icon-sublist-size: var(--sapFontSmallSize) !default;
+  $fd-menu-icon-height: 1.25rem !default; // to account for item top padding of 12px or 0.75rem so 2.75rem-0.75rem-0.75rem = 1.25rem
+  $fd-menu-icon-width: 2.25rem !default; // to account for item right padding of 16px or 1rem so 2.25rem-1rem = 1.25rem
+  $fd-menu-icon-width-compact: 2rem !default;
+  $fd-menu-icon-width-mobile: 2.75rem !default;
+  $fd-menu-icon-color: var(--sapContent_NonInteractiveIconColor) !default;
+  $fd-menu-icon-color-active: var(--sapList_Active_TextColor) !default;
+  $fd-menu-icon-color-selected: var(--sapContent_NonInteractiveIconColor) !default;
+
+  // Menu Title
+  $fd-menu-title-font-size: var(--sapFontSize) !default;
+
+  // Menu SHortcur
+  $fd-menu-shortcut-height: 1.25rem !default;
+  $fd-menu-shortcut-padding-left: 1rem !default;
+  $fd-menu-shortcut-font-size: var(--sapFontSize) !default;
+  $fd-menu-shortcut-color: var(--sapContent_NonInteractiveIconColor) !default;
+
+  // Separator
   $fd-menu-separator-color: var(--sapGroup_ContentBorderColor) !default;
+  $fd-menu-separator-height: 0.0625rem !default;
 
-  $fd-menu-primary-icon-color: var(--sapContent_NonInteractiveIconColor) !default;
-  $fd-menu-primary-icon-color-active-selected: var(--sapList_Active_TextColor) !default;
+  @mixin selected-menu-item() {
+    border-bottom: $fd-menu-item-border-bottom;
+    background-color: $fd-menu-item-background-color-selected;
+
+    .#{$block}__shortcut,
+    .#{$block}__addon-before,
+    .#{$block}__addon-after {
+      color: $fd-menu-icon-color-selected;
+    }
+
+    @include fd-hover() {
+      &:not(:active):not(.is-active) {
+        background-color: $fd-menu-item-background-color-selected-hover;
+      }
+    }
+  }
 
   // BLOCK BASE *******************************************
-  // set all BLOCK reset and baseline styles
   @include fd-reset();
 
   width: 100%;
-  max-width: 20rem;
+  max-width: $fd-menu-max-width;
 
   &--full-width {
     max-width: 100%;
-
-    .#{$block}__link {
-      max-width: 100%;
-    }
   }
 
   &__list,
   &__sublist {
     @include fd-reset();
 
-    border-radius: inherit;
-    margin: 0;
-    padding: 0;
-    list-style: none;
     display: flex;
     flex-wrap: wrap;
     flex-direction: column;
-    -webkit-box-direction: normal;
-    -webkit-box-orient: vertical;
+    list-style: none;
+    box-shadow: $fd-menu-box-shadow;
+    border-radius: $fd-menu-border-radius;
+
+    &--no-shadow {
+      box-shadow: none;
+    }
   }
 
   &__sublist {
     @include fd-reset();
 
-    left: 100%;
-    right: initial;
     position: absolute;
     top: 0;
+    left: 100%;
     z-index: 2;
-    margin: 0.25rem 0 0 -0.25rem; // offset of submenu
+    margin: $fd-menu-sublist-offset;
+    width: 100%; // to fix IE issue
 
     @include fd-rtl() {
       right: 100%;
       left: initial;
       margin: 0.25rem -0.25rem 0 0;
```

Comment:
Since you are moving everything to variables this one can be a var too

Reply:
`0.25rem` should be in var